addons/missileguidance/XEH_PREP.hpp

```diff
@@ -59,6 +59,7 @@ PREP(mwr_onFired);
 PREP(IR_onFired);
 
 // Navigation OnFired
+PREP(los_onFired);
```

Comment:
```suggestion
PREP(navigationType_lineOfSight);
```
I think this is right

Reply:
yeah, `los_onFired` originally was a copy of `pronav_onFired`, but I reduced them to the same file for simplicity